.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           cache: npm
-          node-version: 16
+          node-version: 18
```

Comment:
we just ran https://github.com/gr2m/octoherd-script-bump-node-version-in-workflows/ against the org to use `lts/*` here so that we dont have to keep updating these over time. i'd pull that in here in place of this change

Reply:
Cool, thanks for the hint! Adopted changes from `main`.